app/models/translation.rb

```diff
@@ -34,6 +34,17 @@ def translated_page(page_id, strict)
     xml.to_s
   end
 
+  def translated_tip(tip_id, strict)
+    tip = Tip.find(tip_id)
+    phrases = download_translated_phrases(tip.name)
```

Comment:
for the onesky filename I want to transform the tip name like this:
```
    phrases = download_translated_phrases("tip_${tip.name}.json")
```

Reply:
that will give the json extension which plays nicer with the OneSky web UI (mistake we made with page names using `.xml`), and also "namespace" tip translation files to avoid potential filename collisions.